.github/workflows/lint.yml

```diff
@@ -14,7 +14,15 @@ jobs:
       - name: Checkout 🛎️
         uses: actions/checkout@v2
 
-      - name: Setup Rust ⚙️
+      - name: Verify .rs files 👀
+        uses: tj-actions/verify-changed-files@v11.1
```

Comment:
Why are we using this actions instead of [technote-space/get-diff-action](https://github.com/technote-space/get-diff-action) like we do on other repos as well (i.e. [Desmos](https://github.com/desmos-labs/desmos/blob/master/.github/workflows/test.yml#L54))?

I would prefer if we can stick to the same action throught all the repos if possible, so it's easier to manage for everyone.